### PR TITLE
allow to add raster and vector datasets

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

**Comment by jdroenner** [comment]
[Python: Public Vector-Datasets hinzufügen](https://trello.com/c/gW2YYiDD/1262-python-public-vector-datasets-hinzuf%C3%BCgen)

File: examples/add_public_raster_dataset.ipynb [hunk 1]
```diff
@@ -249,24 +248,26 @@
...
-     "data": {
-      "text/plain": [
-       "688e386b-c9ac-4598-98ea-6d8a9d52f362"
-      ]
-     },
-     "execution_count": 6,
-     "metadata": {},
-     "output_type": "execute_result"
+     "ename": "GeoEngineException",
+     "evalue": "BodyDeserializeError: UUID parsing failed: invalid character: expected an optional prefix of `urn:uuid:` followed by [0-9a-fA-F-], found `t` at 1 at line 1 column 35",
```
Comment (ChristianBeilschmidt) [inline examples/add_public_raster_dataset.ipynb:256]: Your example notebook seems to crash. Maybe you can see why this error occurs.

File: examples/add_public_raster_dataset.ipynb
Comment (ChristianBeilschmidt) [inline examples/add_public_raster_dataset.ipynb:1]: Maybe we should also add an example for adding vectors, then?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: examples/add_public_raster_dataset.ipynb (hunk 1 above)
Comment (jdroenner) [reply examples/add_public_raster_dataset.ipynb]: [fa335ce](https://github.com/geo-engine/geoengine-python/pull/117/commits/fa335ce1b6e64e6351e97718929409cbefd541fb)

File: examples/add_public_raster_dataset.ipynb
Comment (jdroenner) [reply examples/add_public_raster_dataset.ipynb:1]: [fdac65b](https://github.com/geo-engine/geoengine-python/pull/117/commits/fdac65bbecabf78c149a595b28f393eb178c9e6f)

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**